### PR TITLE
Sales eng mp trigger deploy

### PR DESCRIPTION
**Pull request by alexgraham-da**
add trigger management

### CODE REVIEW

File: ui2/src/context/AutomationContext.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <AutomationStateContext.Provider value={{ automations, loading }}>
+      {children}
+    </AutomationStateContext.Provider>
+  );
+};
+
+export function useAutomations() {
+  const context = React.useContext<AutomationState>(AutomationStateContext);
+  if (context === undefined) {
+    throw new Error('useCustomerServices must be used within an AutomationProvider');
```
Comment (alexmatson-da) [inline ui2/src/context/AutomationContext.tsx:53]: Can change the error text here to `useAutomations`

**Review by alexmatson-da — APPROVED** [review]
Looks good to merge after #281

File: ui2/src/pages/login/QuickSetup.tsx [hunk 2]
```diff
@@ -363,16 +407,41 @@ const MarketSetup = (props: {
...
+              )}
               <Table.Cell>
                 <Button
                   className="ghost"
-                  onClick={() => loginUser(dispatch, history, computeCredentials(p))}
+                  onClick={() =>
+                    loginUser(
+                      dispatch,
+                      history,
+                      parties.find(party => party.party === p) || computeCredentials(p)
```
Comment (Sarahb-da) [inline ui2/src/pages/login/QuickSetup.tsx:442]: ah thanks!

**Review by Sarahb-da — APPROVED** [review]
Nice! There will be a small amount of conflicts with what I just merged in, but shouldn't be too bad. I also might re-work the user experience with the Quick Setup modal in general after u merge this in. We have all the major pieces in place now 👍

**Review by brian-weir — APPROVED** [review]

**Pull request merged**